tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -384,4 +425,13 @@ Object.Base {
 		sink	"copier.host.11.1"
 	}
 
+	route."7" {
+		source	"mixin.1.1"
+		sink	"mixout.2.1"
+	}
+
+	route."8" {
+		source	"mixin.13.1"
+		sink	"mixout.2.1"
+	}
```

Comment:
Can I suggest a two-step solution @RanderWang 

a) first transition from copier-copier to using mixing-mixout (still with a single stream)
b) add a new stream to the mixer, and make it conditional so that it can be disabled as needed (same as for topology1).

that would make it easier to proceed with validation, and isolate issues related to the second stream.

Reply:
thanks.  done!

Reply:
@RanderWang I was suggesting the use of two PRs, not two patches, so that we could look at intermediate validation results. One.step.at.a.time.

Reply:
sure, I got it